src/CodeStyle/CSharp/Analyzers/PublicAPI.Shipped.txt

```diff
@@ -7,50 +7,50 @@ Microsoft.CodeAnalysis.CSharp.Formatting.LabelPositionOptions
 Microsoft.CodeAnalysis.CSharp.Formatting.LabelPositionOptions.LeftMost = 0 -> Microsoft.CodeAnalysis.CSharp.Formatting.LabelPositionOptions
 Microsoft.CodeAnalysis.CSharp.Formatting.LabelPositionOptions.NoIndent = 2 -> Microsoft.CodeAnalysis.CSharp.Formatting.LabelPositionOptions
 Microsoft.CodeAnalysis.CSharp.Formatting.LabelPositionOptions.OneLess = 1 -> Microsoft.CodeAnalysis.CSharp.Formatting.LabelPositionOptions
-static Microsoft.CodeAnalysis.CSharp.Formatting.CSharpFormattingOptions.IndentBlock.get -> Microsoft.CodeAnalysis.Options.Option<bool>
```

Comment:
Just namespace changes here - any case, changes to public API files in CodeStyle layer are only for tracking purpose. It does not expose any consumable public APIs.